tests/test_utils.py

```diff
@@ -190,6 +190,20 @@ def insert_text_in_file(file_path: pathlib.Path, tag: str, text: str) -> int:
     return 1
 
 
+def delete_line_in_file(file_path: pathlib.Path, tag: str) -> int:
```

Comment:
Probably better to return boolean

Reply:
changed in several places to return bool